roles/manila/vars/main.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2022 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+_manila_helm_values:
+  endpoints: "{{ openstack_helm_endpoints }}"
+  images:
+    tags: "{{ atmosphere_images | vexxhost.atmosphere.openstack_helm_image_tags('manila') }}"
+  pod:
+    replicas:
+      api: 3
+      data: 3
+      scheduler: 3
+      share: 3
+  conf:
+    manila:
+      oslo_messaging_notifications:
+        driver: noop
+
+  dependencies:
```

Comment:
Shouldn’t this live in the Helm chart?

Reply:
well, i didn't make this as default in the chart and just commented with description.

Reply:
because users can select other driver types or even they can use share service without driver